src/agent/src/policy.rs

```diff
@@ -85,7 +86,7 @@ impl AgentPolicy {
     }
 
     /// Initialize regorus.
-    pub async fn initialize(&mut self, default_policy_file: &str) -> Result<()> {
+    pub async fn initialize(&mut self, policy_file: &str) -> Result<()> {
```

Comment:
@Sumynwa , would this simplified version work too?

```
    /// Initialize regorus.
    pub async fn initialize(&mut self) -> Result<()> {
        if AGENT_CONFIG.log_level.as_usize() >= slog::Level::Debug.as_usize() {
            self.log_file = Some(
                tokio::fs::OpenOptions::new()
                    .write(true)
                    .truncate(true)
                    .create(true)
                    .open(POLICY_LOG_FILE)
                    .await?,
            );
            debug!(sl!(), "policy: log file: {}", POLICY_LOG_FILE);
        }

        let mut default_policy_file = AGENT_CONFIG.policy_file;
        if default_policy_file.is_empty() {
            default_policy_file = POLICY_DEFAULT_FILE.to_string();
        }
        info!(sl!(), "default policy: {default_policy_file}");

        self.engine.add_policy_from_file(default_policy_file)?;
        self.update_allow_failures_flag().await?;
        Ok(())
    }
```

Reply:
Yes @danmihai1 Made the changes.